extensions/sql-database-projects/src/tools/packageHelper.ts

```diff
@@ -65,7 +65,11 @@ export class PackageHelper {
 			if (project) {
 				await this.addPackage(project, packageName, packageVersion);
 			} else {
-				void vscode.window.showInformationMessage(`To use SQL bindings, ensure your Azure Functions project has a reference to ${constants.sqlExtensionPackageName}`);
+				const result = await vscode.window.showInformationMessage(constants.addPackageReferenceMessage, constants.moreInformation);
```

Comment:
You should probably put the message handler in a .then so it doesn't block this from continuing on